canonical/ubuntu.com
# feat: /ceph/install redesign

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1EmOtHjuEMKhVLQA7GwyrKfCzOKfc8UCEyh-p04-PxzA/edit?tab=t.0) and [design](https://www.figma.com/design/ulUyOFdQ7CNy0whuJ4S2FM/Ceph-Bubble?node-id=16-4455&node-type=frame&m=dev)
- Drive by:
  - Fixed the djlint toggle setting not placed correctly on `/_latest-blogs_download.html`

## QA

- View the site locally in your web browser at: https://ubuntu-com-14511.demos.haus/ceph/install
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16347
